packages/config-utils/standalone/services/default/chrome.js

```diff
@@ -39,7 +39,7 @@ module.exports.registerChrome = ({ app, chromePath, keycloakUri, https, proxyVer
     // Unfortunately we have to alter the contents of ALL text/html responses like akamai
     // This means modifying the response object directly.
     app.use((req, res, next) => {
-        const ext = path.extname(req.url);
+        const ext = path.extname(req.url.replace(/\?.*/, ''));
```

Comment:
Maybe I would extract this to a separate function and call it something like "removeUrlQuery" so everybody know what it does without solving the regex. But this is just a minor nitpick. 🌵

Reply:
Or you can just extract the regex and name it.

Reply:
If I end up needing it again I'll extract it!